lib/v2/components/search_user/search_user_widget.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/components/search_user/components/search_result_row.dart';
+import 'package:seeds/v2/components/search_user/interactor/search_user_bloc.dart';
+import 'package:seeds/v2/components/search_user/interactor/viewmodels/search_user_events.dart';
+import 'package:seeds/v2/components/search_user/interactor/viewmodels/search_user_state.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/datasource/remote/model/member_model.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+class SearchUserWidget extends StatelessWidget {
+  final ValueSetter<MemberModel> resultCallBack;
```

Comment:
This callback is not used?

Reply:
No yet, next pr ill attach the callbacks logic and the UI